SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/Digital/PPMU.cs

```diff
@@ -157,6 +201,35 @@ public static void ForceCurrent(
             });
         }
 
+        /// <summary>
+        /// Forces current on the target pin(s).
+        /// </summary>
+        /// <param name="sessionsBundle">The <see cref="DigitalSessionsBundle"/> object.</param>
+        /// <param name="settings">The settings to use.</param>
+        public static void ForceCurrent(this DigitalSessionsBundle sessionsBundle, PPMUSettings settings)
+        {
+            sessionsBundle.Do(sessionInfo =>
+            {
+                settings.OutputFunction = PpmuOutputFunction.DCCurrent;
+                sessionInfo.Session.Force(sessionInfo.PinSetString, settings);
+            });
+        }
+
+        /// <summary>
+        /// Forces current on the target pin(s).
```

Comment:
Suggest changing to: "Forces current using specified site-unique ppmu settings."